docs/api/api/api.txt

```diff
@@ -1847,6 +1847,36 @@ PUT /message/?project=<project>&package=<package>
 XmlBody :message
 XmlResult: message
 
+== Status Checks
+  WARNING: This API is currently in beta and unstable
+
+=== Repositories
+
+<project>: The name of a project, e.g. home:user1
+<repository>: The name of a repository of the project, e.g. openSUSE_Tumbleweed
+<uuid>: The unique identifier you have received for the Repository
```

Comment:
this sounds like $project/$name location, but it is actually the iteration of the repository. So, identifier for the iteration or meta data revision is maybe more exact.

Reply:
Why does it matter what it really is? You get a uuid, you send a uuid. This is user documentation right? How about 

> <uuid>: The build id you have received for the repository

build id is what it is called in the `Event`.